### DISCUSSION TITLE
Clarification about schedules

### DISCUSSION THREAD

**Discussion by sprat** [renovatebot/renovate#32080] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

I have many repositories with more or less the same configuration:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:recommended", ":automergeAll"]
}
```
Or:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:recommended", ":maintainLockFilesWeekly", ":automergeAll"]
}
```
However, in the Mend.io hosted app, some of my repositories have a "hourly" schedule whereas other have a "daily" schedule (as shown in the following screenshots:
![image](https://github.com/user-attachments/assets/a8916f70-58d4-46ca-84ec-335daaec4582)
Or:
![image](https://github.com/user-attachments/assets/208aaae8-e63b-47ca-a985-a269b1e9639e)

What could explain this difference in the schedules?

**Comment by rarkins** [comment]
The schedule you see in the mend portal represents how often Mend enqueues a job to be run. It's not related at all to the schedule you configure in the repo config

**Reply by sprat** [reply]
And so, why Mend does not enqueue jobs at the same rate for all the repositories? It is related to the dependencies used in the repo?

**Reply by rarkins** [reply, marked as answer]
Go to the org view page and look for the "status" of each repo. Some will be onboarding, some onboarded, and activated. Generally activated are enqueued hourly while onboarded (which means config exists but no PRs merged yet) will be daily.

**Reply by sprat** [reply]
Ok thanks, it's clear now!

**Reply by rarkins** [reply]
Essentially this is just our way to balance load by not running repos as regularly as others if they don't seem to need it

**Reply by sprat** [reply]
It makes sense, that's what I suspected also ;)
